ui/src/app/views/signed-in/component.ts

```diff
@@ -163,8 +163,8 @@ export class SignedInComponent implements OnInit, OnDestroy {
     return this.locationService.path() === '/admin/review-workspace';
   }
 
-  get reviewIdActive(): boolean {
-    return this.locationService.path() === '/admin/review-id-verification';
+  get userAdminActive(): boolean {
+    return this.locationService.path() === '/admin/user';
```

Comment:
nit: maybe admin/users with the plural is slightly better? I won't argue if you keep as-is.

Reply:
I think it is the admin user page. I like the authority/page being called `User Admin`, so want the path to reflect that. If we find out later, happy changing it.